hw/ip/pwrmgr/doc/_index.md

```diff
@@ -119,6 +119,19 @@ If none of these exception cases are matched for low power entry, the fast FSM t
 For reset requests, fall through and aborts are not checked and the system simply resets directly.
 Note in this scenario the slow FSM is not requested to take over.
 
+### ROM Integrity Checks
+
+The power manager coordinates the [start up ROM check]({{< relref "hw/ip/rom_ctrl/doc/_index.md#the-startup-rom-check" >}}) with `rom_ctrl`.
+
+The power manager sends an indication to the `rom_ctrl` to begin performing integrity checks after every reset.
```

Comment:
Nit: Maybe "After every reset, the power manager..." Otherwise it sounds a bit like the signal is telling `rom_ctrl` to do something after every reset.

Reply:
sounds good, let me update.